### ISSUE TITLE
Idea: NodeSchool Online

### ISSUE THREAD

**Issue by maxogden** [nodeschool/organizers#233]
This came out of a discussion at NodeConf this week.

The idea is to create a new event type -- a "NodeSchool Online" event, in which we:

- get a mentor or two to commit to hosting a specific workshop in a Google Hangout at a specific time
- e.g. "Substack will be hosting the Stream Adventure workshop Monday at 4PM PST"
- Online events will have 5 - 10 open slots available, anyone can sign up for these slots
- The event itself will be very similar to the in-person events, but it will happen over Google Hangouts
- If attendees are OK with it we can stream the hangouts using Hangouts On Air to [our new youtube channel](https://www.youtube.com/channel/UCzUA8Ch_NqTDzdgynhnLJFg)
- we can advertise upcoming online workshops on the nodeschool.io homepage
- we can reach out to specific members of the community to be guest mentors (like Node core contributors, module authors, etc)
- anyone should be able to offer to host an online workshop

Questions

- What do you all think?
- Does anyone wanna do a mockup of what it would look like to add a new section for this to the nodeschool homepage?
- How should we handle both submitting a new workshop (as a host) and signing up for a workshop (as an attendee)?
- Is first-come-first-served a good system for attendee signups?

**Comment by bleedingxedge** [comment]
This sounds AWESOME

I can do the mockup for the new section of the homepage (if anyone else wants to help, please do)
Maybe something like eventbrite or meetup could work for the hosting/attending workshops.
Yeah, FCFS sounds good.

**Comment by jedireza** [comment]
This sounds cool. Twitch.tv may be worth reviewing. I've seen some people use twitch for their live events and then upload the session to a youtube channel afterwards.

**Comment by tzmanics** [comment]
I'm really excited about this idea because it helps get more people involved that don't have events in their area. I also really like the idea of reaching out to core contributors and module authors to be guest mentors.

FCFS sounds good but I'm wondering if there should also be a small waiting list. If the event is set up with Meetup it's easy to cap the attendance and make a waiting list.

**Comment by tako-black** [comment]
This sounds very nice! I want to host a channel of workshop that I create.
My workshop, [learnyoureact](https://github.com/tako-black/learnyoureact) has been used by a lot of people around the world.
I'm very happy, but I'm very sad that I cannot talk to users and get feedbacks.
If creator of workshop get feedbacks, the workshop will be more nice!
So, I agree to your opinion!

**Comment by milesibastos** [comment]
This sounds AWESOME! Do you have some news?!?

**Comment by JonathanWatkins** [comment]
This is a great idea. Can we make it happen?

**Comment by smallhaxe** [comment]
Hangouts and Twitch both have built in support for exporting to Youtube (nice).

Talky is another option. While the basic version does not have recording and broadcast features, it also does not require users to register/log into an account in order to participate fully via voice / video / screenshare / chat (nice). There's also [about.talky.io/core](https://about.talky.io/core). (Has anyone used this?)

Could we treat **NodeSchool Online** as a chapter, and organize hosts, workshops, and mentors via issues within that chapter?

First-come-first-served (actually joining the session) seems to me to be the best approach to attendees, at least to start.

Whatever the technology and approach, this is a great idea!

**Comment by yoshuawuyts** [comment]
cc/ @jpwesselink what was that live streaming node school event you did a while back?

**Comment by jpwesselink** [comment]
@yoshuawuyts Yes there is! It started out as an experiment with Twitch during Intl day, but it's starting to become a recurring broadcast every Sunday afternoon (CET-ish time). We lovingly dubbed it ['NodeSchool Space Station'](https://github.com/nodeschool/amsterdam/issues/15)

**Comment by martinheidegger** [comment]
@maxogden @jpwesselink ... what is the status of this?

**Comment by hollomancer** [comment]
We'd be interested in championing this!
